packages/shared/src/components/icons/Home/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Icon, { IconProps } from '../../Icon';
+import OutlinedIcon from './outlined.svg';
+import FilledIcon from './filled.svg';
+
+const HomeIcon: React.VFC<IconProps> = ({
```

Comment:
Kindly check my concern regarding the `props` for easier maintenance. Tagging for visibility. Thank you!